client/pages/user-page.jsx

```diff
@@ -35,11 +39,50 @@ class UserPage extends React.Component {
         this.setState({ chefs: result });
       })
       .catch(err => console.error(err));
+
+    fetch('/api/userProfile/reviews', {
+      headers: {
+        'X-Access-Token': token
+      }
+    })
+      .then(response => response.json())
+      .then(result => {
+        this.setState({ reviews: result });
+      })
+      .catch(err => console.error(err));
+  }
+
+  handleDeleteClick(event) {
+    const token = window.localStorage.getItem('final-project-jwt');
+    const chefId = event.target.getAttribute('chefId');
+    fetch(`/api/userProfile/${chefId}`, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Access-Token': token
+      }
+    })
+      .then(response => response.json())
+      .then(() => {
+        const newChefArray = this.state.chefs.filter(chef => {
+          return !chefId.includes(chef.chefId);
+        });
```

Comment:
Ok, this is good, but kind of confusing. Instead convert the string to a number and do a comparison

```js
    const chefId = Number(event.target.getAttribute('chefId'));
    // ... fetch
        const newChefArray = this.state.chefs.filter(chef => {
          return chefId !== chef.chefId;
        });
```